migrate-to-github/README.md

```diff
@@ -0,0 +1,77 @@
+Check the [what-to-migrate.md]() file first, please.
+
+### How we want to do it
+
+We will use the XXX tool (not released yet).
+  - still thinking about its name :D I think about two possibilities:
+    - `volagit` or `forgit` -> these are just random names containing `git`... I think
+       about them because they sounds cool :D
+
+### What tools it will use
+
+- [Ogr](https://github.com/packit/ogr)
+  - A tool which wraps Pagure, Gitlab and Github API into one API
+  - There's 95% of API calls we need to use 
+- [GitPython](https://gitpython.readthedocs.io/en/stable/)
+  - A tool which wraps git cli to a Python api
+  - only when Github, Gitlab API doesn't allow something (e.g. `git reset`). In other cases Ogr
+    includes all the stuff we need.
+- That's it
+  
+### What we need to use to achieve the goals in [what-to-migrate.md]()
+
+- Create a bot account which will transfer the PRs/Issues to Github
+- Create Github mirror of Copr project
+  - Copr migrated from Github to Pagure, where it had already some 30 PRs. The project
+    on Pagure did not migrate these PRs and started on PR id 2 -> the previous 30 PRs
+    are lost from context. But in the current Github mirror these PRs are there and
+    we need to create a new clean mirror to match all the Pagure IDs.
```

Comment:
I think we can skip the first 30 ids while doing migration.  There's no good solution;  some references to those IDs might already be confusing because we are not sure if they refer to the github repo or to pagure one.
That said, I'd personally prefer the fedora-copr/copr repo we already have.

Reply:
the mirror repo still can be deleted and then recreated with no PR or Issues
another problem is that I can't see Issues in the fedora-copr/copr repo. They seems to be disabled but there may be some issues archived

Reply:
but yes there's no good solution - we can ignore the first N issues/PRs from pagure... which will bring mess to those N IDs or we can delete the old PRs and Issues on GH (if they are less relevant to us than the Pagure's PRs/Issues)

Reply:
We'll enable issues again before the move I think.  It looks like there are no issues though (even PRs were rare in this project those days).  But the project is somewhat under users' radars, removing and recreating it would be a pity.

Reply:
> I think we can skip the first 30 ids while doing migration.

Agreed. If I understand it correctly, we will lose the first 30 PRs/issues from Pagure, that's no biggie, out of ~2k of them. Also, they are 5 years old and AFAIK nobody referenced them for years, so it won't affect our day-to-day lives.

> I'd personally prefer the fedora-copr/copr repo we already have.

Me too, we already have 53 stars and 22 forks and I wouldn't want to cause unnecessary confusion among those people. Also, we AFAIK have some configurations CI within the project that would have to be done again.